arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+              "$ref": "#/definitions/RuleResourceCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-odata": "#/definitions/RuleResource"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/autoscalesettings": {
+      "get": {
+        "description": "Lists the autoscale settings for a resource group",
+        "tags": [
+          "Autoscale"
+        ],
+        "operationId": "AutoscaleSettings_List",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource name."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request for a list of autoscale settings",
+            "schema": {
+              "$ref": "#/definitions/AutoscaleSettingResourceCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-odata": "#/definitions/AutoscaleSettingResource"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/autoscalesettings/{autoscaleSettingName}": {
+      "patch": {
+        "description": "Updates and autoscale setting",
+        "tags": [
+          "Autoscale"
+        ],
+        "operationId": "AutoscaleSettings_Update",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource name."
+          },
+          {
+            "name": "autoscaleSettingName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The autoscale setting name."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AutoscaleSettingCreateOrUpdateParameters"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created autoscale setting"
```

Comment:
missing schema for responses
check this link https://msdn.microsoft.com/en-us/library/azure/dn931928.aspx